wandb/sdk/internal/internal_api.py

```diff
@@ -1932,7 +1931,7 @@ def upload_file(
             is_aws_retryable = (
                 "x-amz-meta-md5" in extra_headers
                 and status_code == 400
-                and "RequestTimeout" in response_content
```

Comment:
This was broken. On `main`, if I force this code path to get taken...

```
$ python3 -c 'import wandb; wandb.init()'
<...snip...>
wandb: ERROR Error uploading "wandb-metadata.json": CommError, a bytes-like object is required, not 'str'
wandb: ERROR Error uploading "requirements.txt": CommError, a bytes-like object is required, not 'str'
wandb: ERROR Error uploading "wandb-summary.json": CommError, a bytes-like object is required, not 'str'
wandb: ERROR Error uploading "config.yaml": CommError, a bytes-like object is required, not 'str'
```

`debug-internal.log` confirms that this is because `response_content` is a `bytes`, but `"RequestTimeout"` is a `str`.

Reply:
these are great catches!